src/vivarium_public_health/dataset_manager/hdf.py

```diff
@@ -15,25 +15,37 @@
 
 
 def touch(path: str, append: bool):
-    """Creates an hdf file if necessary or errors if trying to append to a non-existent file.
+    """Creates an hdf file or wipes an existing file if necessary.
+
+    If the file exists and append is not true, the existing file
+    will be destroyed. If the file exists and append is true, touch
+    will do nothing.
 
     Parameters
     ----------
     path :
         The string path to the hdf file.
     append :
-        Whether or not we want to append to an existing file.
+        Whether an existing artifact should be preserved and appended to
 
     Raises
     ------
     FileNotFoundError
         If attempting to append to a non-existent file."""
+
     path = Path(path)
 
-    if append and not path.is_file():
-        raise FileNotFoundError("You indicated you want to append to an existing artifact "
-                                f"at {path} but no such artifact exists.")
-    elif not append:
+    if path.is_file() and append:
+        pass
+    if path.is_file() and not append:
```

Comment:
Why do you need this branch if it does the same thing as the else?

Reply:
The else will wipe the file by opening in write mode, is_file() and append actually does nothing. I included it just to enumerate all the branches for clarity

Reply:
I'm talking about the is_file and not append branch though - that seems like it also wipes the file?